node-backend/lib/ai.mjs

```
@@ -43,39 +46,53 @@ export class AIService {
+        case 'gemini_cli':
+          return new GeminiCliProvider(await withDetectedCommand(getCfg('geminiCli', 'gemini_cli'), 'gemini'), secrets);
+        case 'claudeCode':
+        case 'claude_code':
+        case 'claude':
+          return new ClaudeCodeProvider(await withDetectedCommand(getCfg('claude', 'claude_code'), 'claude'), secrets);
+        case 'openAiCli':
+        case 'openai_cli':
+        case 'openai':
+          return new OpenAiCliProvider(await withDetectedCommand(getCfg('openAiCli', 'openai_cli'), 'codex', 'openai'), secrets);
+        default:
+          const customClis = settings.customClis || settings.custom_clis || [];
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)